editor/src/core/shared/template-path.ts

```diff
@@ -219,8 +219,8 @@ export function asStatic(path: InstancePath): StaticInstancePath {
   return path as StaticInstancePath
 }
 
-export function isScenePath(path: TemplatePath): path is ScenePath {
-  return (path as any).type === 'scenepath'
+export function isScenePath(path: unknown): path is ScenePath {
+  return (path as any)?.type === 'scenepath'
```

Comment:
Turns out this magical question mark saves us from having to write
```
typeof path === 'object' && path != null && (path as any).type ===
```

Who knew?! Balazs apparently :grinning: